manageiq-operator/pkg/apis/manageiq/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
LOL - did you add this or did something do this for you?

Reply:
That was added by `gofmt -w .`

Reply:
Oh good, this conflicts with operator-sdk generated code...